test/functional/test_framework/wallet.py

```diff
@@ -77,3 +77,30 @@ def send_self_transfer(self, *, fee_rate=Decimal("0.003"), from_node, utxo_to_sp
         assert_equal(tx_info['vsize'], vsize)
         assert_equal(tx_info['fee'], fee)
         return {'txid': txid, 'wtxid': tx_info['wtxid'], 'hex': tx_hex}
+
+    def send_multi_self_transfer(self, *, fee_rate=Decimal("0.03"), from_node, utxo_to_spend=None, big_txouts=None):
+        """Create multiple tx with the specified fee_rate. Fee may be exact or at most one satoshi higher than needed."""
+        self._utxos = sorted(self._utxos, key=lambda k: k['value'])
+        utxo_to_spend = utxo_to_spend or self._utxos.pop() # Pick the largest utxo (if none provided) and hope it covers the fee
+        send_value = satoshi_round(utxo_to_spend['value'] - fee_rate)
+        fee = utxo_to_spend['value'] - send_value
+        assert send_value > 0
+
+        tx = CTransaction()
+        tx.vin = [CTxIn(COutPoint(int(utxo_to_spend['txid'], 16), utxo_to_spend['vout']))]
+        tx.vout = [CTxOut(int(send_value * COIN), self._scriptPubKey)]
+        tx.wit.vtxinwit = [CTxInWitness()]
+        tx.wit.vtxinwit[0].scriptWitness.stack = [CScript([OP_TRUE])]
+        tx_hex = tx.serialize().hex()
+
+        if(big_txouts is not None):
+            for txout in big_txouts:
+                tx.vout.append(txout)
+            tx_hex = tx.serialize().hex()
+            txid = from_node.sendrawtransaction(tx_hex, 0)
+        else:
+            txid = from_node.sendrawtransaction(tx_hex)
+        self._utxos.append({'txid': txid, 'vout': 0, 'value': send_value})
+        tx_info = from_node.getmempoolentry(txid)
+        assert_equal(tx_info['fee'], fee)
```

Comment:
Just want to confirm that we no longer need a `vsize` assertion. I guess we don't know the `vsize` or don't care since we are passing an absolute fee and not calculating the fee based on vsize*fee_rate?

Reply:
Yes thats what i realized. Also the `vsize` changed (from 96 to 67552) while making big txs  and therefore checking it with a constant value of 96 didn't really make sense. I can however, add an assertion to check big tx `vsize` since they're all the same

Reply:
I'm not really sure tbh. But we may be able to get rid of all of this, re one of my other comments in this review